src/groups/mwc/mwcio/mwcio_ntcchannel.h

```diff
@@ -371,7 +372,7 @@ class NtcChannel : public mwcio::Channel,
     /// Shutdown this channel, and cancel all pending read requests (but do
     /// not invoke them).  Pass the specified `status` to any registered
     /// `CloseFn`s.
-    void close(const Status& status) BSLS_KEYWORD_OVERRIDE;
+    void close(const Status& status = Status()) BSLS_KEYWORD_OVERRIDE;
```

Comment:
https://github.com/bloomberg/blazingmq/blob/50018e73fece05e5bebcb2f3ad3b1e31d49e1c17/src/groups/mwc/mwcio/mwcio_channel.h#L193
The interface has default for status